ddtrace/contrib/jinja2/patch.py

```diff
@@ -17,7 +18,7 @@
 config._add(
     "jinja2",
     {
-        "service_name": get_env("jinja2", "service_name"),
+        "service_name": os.getenv("DD_JINJA2_SERVICE_NAME"),
```

Comment:
note: we should deprecate this. I don't see why someone needs to set a service name for `jinja2`

Reply:
(don't make this change here, just calling it out)